docs/contracts/functions.rst

```diff
@@ -332,15 +332,20 @@ Below you can see an example of a Sink contract that uses function ``receive``.
 Fallback Function
 =================
 
-A contract can have at most one ``fallback`` function, declared using ``fallback () external [payable]``
```

Comment:
I think the docs should mention that `fallback` and `receive` can be virtual and have modifiers. It might not be obvious.

Reply:
Clarified for both the fallback and the receive function.